peer-feedback.md

```diff
@@ -0,0 +1,27 @@
+# Peer Feedback
+
+We have a developing system of peer feedback to set self-development goals and receive feedback over the course of the year. Around Quarterly Planning meetings, Semi-Annual Retreats, and the Members meetings we have checkins.
+
+## Pair / pod folks for feedback (have a random distribution below)
+
+1. 5 minutes to reflect on work over last year and areas to identify: 
+    - at least 1 thing you did well
+    - 1-3 things you'd like to improve
+2. 15 mins share with your partner
+    - before you start, review this handout on feedback: https://www.seedsforchange.org.uk/feedback.pdf
```

Comment:
```suggestion
    - before you start, review this handout on feedback, together if needed: https://www.seedsforchange.org.uk/feedback.pdf
```

just to be explicit that it's ok to take time to do it together if wasn't time before, since sometimes ppl arrive without review time 🙂